buildpack.md

```diff
@@ -295,23 +302,27 @@ At the end of each individual buildpack's build phase:
 | `<app>/.profile`       | [†](README.md#linux-only) Bash-formatted script sourced by shell before launch
 | `<app>/.profile.bat`   | [‡](README.md#windows-only) BAT-formatted script sourced by shell before launch
 
+TODO: move this after analyze
```

Comment:
Doing this now would make the diff impossible to decipher.

Reply:
Actually... this isn't needed. What we are calling "analysis" all happens in the restore phase now.